tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -38,15 +38,23 @@ sub run {
 
     #	bootloader options and set probe foreign OS, timeout
     assert_and_click 'yast2-bootloader_bootloader-options';
+
+    wait_still_screen 10;
+    send_key 'alt-t';    # Import untrusted GnuPG key -- workaround
+    wait_still_screen 10;
+
     assert_screen 'yast2-bootloader_bootloader-options-switched';
     send_key 'alt-b';
     wait_still_screen 3;
     send_key 'alt-t';
     type_string '16';
 
     #	default boot section
+    for (1 .. 2) { send_key 'alt-f10' }
     assert_and_click 'yast2-bootloader_default-boot-section';
-    assert_screen 'yast2-bootloader_default-boot-section_tw';
+    record_soft_failure('bsc#1191112 - Resizing window as workaround for YaST content not loading');
+    send_key_until_needlematch('yast2-bootloader_default-boot-section_tw', 'alt-f10', 9, 2);
```

Comment:
The problem with sending alt-f10 is that we en-up with fullscreen as here http://d218.qam.suse.de/tests/100#step/yast2_instserver/15. Not the end of the workd, but then we may need to maintain two sets of needles, one for full screen and one for windowed.

Reply:
Then it creates random failures as we saw here https://openqa.suse.de/tests/9289934#step/iscsi_client/77. Not sure how to tackle this.

Reply:
Can we send alt-f10 10 times?

Reply:
@jknphy it sends alt-f10 ten times, not nine, despite what the argument says, check https://progress.opensuse.org/issues/107749

Reply:
Ah ok,  so I suspect this happens just before when we are using `for (1 .. 2) { send_key 'alt-f10' },` no ? I guess the two send_key are sent too fast, then the window stays full screen. I would suggest to put the same send_key_until_needlematch also there.

Reply:
> Ah ok, so I suspect this happens just before when we are using `for (1 .. 2) { send_key 'alt-f10' },` no ? I guess the two send_key are sent too fast, then the window stays full screen. I would suggest to put the same send_key_until_needlematch also there.

I see that here https://openqa.nue.suse.com/tests/9386473#step/installation/59, we en-up full screen while using send_key_until_needlematch, so I was wrong, resolving conversation.

Reply:
This being said, it means the code works, just those fullscreen needles should be removed, and we should create more specific ones that are not fullscreen.

Reply:
I added send_key_until_needlematch, so I think it's fixed.

Reply:
> This being said, it means the code works, just those fullscreen needles should be removed, and we should create more specific ones that are not fullscreen.

ok, I will revert the code.

Reply:
I am saying that the code works, why do you want to revert it ?